src/prep_helpers.py

```
@@ -670,9 +671,24 @@ async def process_trackers_and_torrent(
     if not any(meta.get(id_type) for id_type in hash_ids + tracker_ids) and not meta.skip_trackers and not meta.edit:
-        await client.get_pathed_torrents(str(meta.path), meta)
+        reuse_torrent_path = await client.find_existing_torrent(meta)
+        if reuse_torrent_path:
+            meta.reuse_torrent_path = reuse_torrent_path
+            if meta.subtitle_files and client._torrent_includes_all_local_subtitles(reuse_torrent_path, meta):
+                meta.subs_reuse_torrent_path = reuse_torrent_path
+            else:
+                meta.base_reuse_torrent_path = reuse_torrent_path
+            try:
+                meta.infohash = Torrent.read(reuse_torrent_path).infohash
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)